.pipelines/singletenancy/dualstack-overlay/dualstackoverlay-e2e-job-template.yaml

```diff
@@ -40,28 +40,28 @@ stages:
           hostport: true
           service: true
 
-      - job: cleanup
-        displayName: "Cleanup"
-        dependsOn:
-          - ${{ parameters.name }}
-          - "cni_k8se2e"
-          - "dualstackoverlay_e2e"
-        pool:
-          name: $(BUILD_POOL_NAME_DEFAULT)
-        condition: always()
-        steps:
-          - task: AzureCLI@1
-            inputs:
-              azureSubscription: $(AZURE_TEST_AGENT_SERVICE_CONNECTION)
-              scriptLocation: "inlineScript"
-              scriptType: "bash"
-              addSpnToEnvironment: true
-              inlineScript: |
-                set -e
-                echo "Deleting cluster"
-                make -C ./hack/aks azcfg AZCLI=az
-                make -C ./hack/aks set-kubeconf AZCLI=az CLUSTER=${{ parameters.clusterName }}-$(make revision)
-                make -C ./hack/aks down AZCLI=az SUB=$(SUB_AZURE_NETWORK_AGENT_TEST) CLUSTER=${{ parameters.clusterName }}-$(make revision)
-                echo "Cluster and resources down"
-            name: "CleanupDualStackOverlaycluster"
-            displayName: "Cleanup DualStack Overlay Cluster"
\ No newline at end of file
+      # - job: cleanup
+      #   displayName: "Cleanup"
+      #   dependsOn:
+      #     - ${{ parameters.name }}
+      #     - "cni_k8se2e"
+      #     - "dualstackoverlay_e2e"
+      #   pool:
+      #     name: $(BUILD_POOL_NAME_DEFAULT)
+      #   condition: always()
+      #   steps:
+      #     - task: AzureCLI@1
+      #       inputs:
+      #         azureSubscription: $(AZURE_TEST_AGENT_SERVICE_CONNECTION)
+      #         scriptLocation: "inlineScript"
+      #         scriptType: "bash"
+      #         addSpnToEnvironment: true
+      #         inlineScript: |
+      #           set -e
+      #           echo "Deleting cluster"
+      #           make -C ./hack/aks azcfg AZCLI=az
+      #           make -C ./hack/aks set-kubeconf AZCLI=az CLUSTER=${{ parameters.clusterName }}-$(make revision)
+      #           make -C ./hack/aks down AZCLI=az SUB=$(SUB_AZURE_NETWORK_AGENT_TEST) CLUSTER=${{ parameters.clusterName }}-$(make revision)
+      #           echo "Cluster and resources down"
+      #       name: "CleanupDualStackOverlaycluster"
+      #       displayName: "Cleanup DualStack Overlay Cluster"
```

Comment:
this was probably commented this for test purposes, but please add it back before merging